flycheck.el

```diff
@@ -7780,6 +7781,24 @@ See URL `http://lesscss.org'."
           line-end))
   :modes less-css-mode)
 
+(flycheck-define-checker llvm-llc
+  "Flycheck LLVM IR checker using llc.
+
+See URL `http://llvm.org/docs/CommandGuide/llc.html'."
+  :command ("llc" "-o" null-device source)
+  :error-patterns
+  ((error line-start
+          ;; llc prints the executable path
+          (zero-or-one (minimal-match (one-or-more not-newline)) ": ")
```

Comment:
I think `flycheck-rx-to-string` translates `(filename)` to `(minimal-match (one-or-more not-newline))`, so I thought would be an equivalente to the executable path. I don't have a windows machine to verify this regexp though.